stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -320,7 +320,7 @@ public class ChatClient internal constructor(
     }
 
     public fun subscribe(
-        listener: (event: ChatEvent) -> Unit,
+        listener: ChatEventListener<ChatEvent>,
```

Comment:
We didn't keep the old declarations in this class

Reply:
Added the old function back and deprecated it